README.TESTING

```diff
@@ -1,181 +0,0 @@
-[IMPORTANT NOTICE]
```

Comment:
Instead of removing README.TESTING, it might be sensible to keep the file but to only refer to http://qa.php.net/write-test.php (like we did in 1e9e7d4dc4949b67908e55ab3f3e44528923c029).

Reply:
To make it even more clear where to find this particular information (because now this wouldn't be found so easily - I agree), we can maybe also add additional paragraph and a link to qa.php.net docs in the main README.md instead?

Reply:
Yeah, adding a link to README.md seems fine.

Reply:
Great. I really think that avoiding this file in the root repo is better choice. I've added a testing introductory paragraph/section to README.md. Now the main README.md also needs some updates and minor fine tuning but this would complicate this pull request out of scope and I would rather do that in a separate pull request. Because the installation chapter - the most important one is missing there. But again, this would be done in a separate pull request because it concerns also Unix build system changes (very minor ones)... Thanks. :)